### ISSUE TITLE
[Profile flow] Block/unblock user

### ISSUE THREAD

**Issue by caybro** [status-im/status-desktop#13522]
Block user:
![image](https://github.com/status-im/status-desktop/assets/5377645/0df4b504-22b9-4a72-b798-74568372d41e)

- Blocked user profile pictures and bios should not be shown.
- Additional (optional) auto actions to also "Remove contact" and "Cancel ID verification request" (if any)

Unblock user:
![image](https://github.com/status-im/status-desktop/assets/5377645/4e2b10c4-d111-4386-becf-4568a92176f6)

Figma: https://www.figma.com/file/ibJOTPlNtIxESwS96vJb06/%F0%9F%91%A4-Profile-%7C-Desktop?type=design&node-id=2460-197035&mode=design&t=qVMy2WcrL4fvxnwK-0

Hiding profile pic/bio reported as a separate issue: https://github.com/status-im/status-desktop/issues/13661
